ans-abs/methodology.tex

```diff
@@ -1,6 +1,6 @@
 \section{Methodology}
 
-In this work we use and extend the methodology described in Baker et. al using the \texttt{RAVEN} framework \cite{baker_optimal_2018,alfonsi_raven_2016}. This methodology allows for empirical quantification of energy system flexibility. Baker et. al used a fixed amount of nuclear power production from a small modular reactor (300 MWe) and varied the penetration of VRE in the form of wind power. They also included grid flexibility in the form of a fixed amount of battery storage. UIUC has wind and solar power purchase agreements that fix the amount of VRE the campus receives. Rather than fixing the nuclear power production, this methodology will be extended to search for the optimal size of a micro-reactor for the campus. \\
+In this work we use and extend the methodology described in Baker et. al using the \texttt{RAVEN} framework \cite{baker_optimal_2018,alfonsi_raven_2016}. This methodology allows for empirical quantification of energy system flexibility. Baker et. al used a fixed amount of nuclear power production from a small modular reactor (300 MWe) and varied the penetration of VRE in the form of wind power. They also included grid flexibility in the form of a fixed amount of battery storage. UIUC has wind and solar power purchase agreements that fix the amount of VRE the campus receives. Rather than fixing the nuclear power production, this methodology will be extended to search for the optimal size of a micro-reactor for the campus. Additionally, the reactor will serve as baseload for steam demand with excess thermal power diverted to generate electricity.\\
```

Comment:
```suggestion
In this work we use and extend the methodology described in Baker et. al using the \texttt{RAVEN} framework \cite{baker_optimal_2018,alfonsi_raven_2016}.
This methodology allows for empirical quantification of energy system flexibility.
Baker et. al used a fixed amount of nuclear power production from a small modular reactor (300 MWe) and varied the penetration of VRE in the form of wind power.
They also included grid flexibility in the form of a fixed amount of battery storage. UIUC has wind and solar power purchase agreements that fix the amount of VRE the campus receives.
Rather than fixing the nuclear power production, this methodology will be extended to search for the optimal size of a micro-reactor for the campus.
Additionally, the reactor will serve as baseload for steam demand with excess thermal power diverted to generate electricity.\\
```